snap/snapcraft.yaml

```diff
@@ -60,18 +59,11 @@ apps:
       - avahi-observe
       - browser-support
       - cups-control
-      - gsettings
       - opengl
-      - home
       - desktop
       - desktop-legacy
-      - mount-observe
       - network
       - network-bind
-      - network-observe
-      - password-manager-service
```

Comment:
Looks like you have a separate PR for this one so I'm going to rm it from this one!

Reply:
Oh fair but it was fine to be her too :)